pyepal/plotting/__init__.py

```diff
@@ -177,16 +234,17 @@ def make_jointplot(  # pylint:disable=invalid-name
     Args:
         y (np.array): array with the objectives (measurements)
         palinstance (PALBase): "trained" PAL instance
-        labels (Union[List[str], None], optional): [description]. Defaults to None.
-        figsize (tuple, optional): [description]. Defaults to (8.0, 6.0).
+        labels (Union[List[str], None], optional): Labels for each objective.
+        Defaults to "objective [index]".
+        figsize (tuple, optional): Figure size for joint plot. Defaults to (8.0, 6.0).
 
     Returns:
         fig
     """
 
     num_targets = y.shape[1]
     fig, ax = plt.subplots(  # pylint:disable=invalid-name
-        num_targets, num_targets, figsize=figsize
+        num_targets, num_targets, figsize=figsize, tight_layout=True
```

Comment:
something we do not have to fix here but In the function above the figsize argument behaves differently

Reply:
yea i thought it made more sense to set the figsize individually for that function and have it auto-adjust by multiplying it based on the number of objectives. probably not good if we want to be consistent with the other function. it didn't make sense to do that for this function since the figure size for each individual subplot will be very small

Reply:
ok, we can maybe describe these behaviors in the docstrings

Reply:
ah it is already there, sorry!